### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by deanwetherby**

### CODE REVIEW

**Comment by deanwetherby** [comment]
Lots of test commits trying to figure out the issue.

File: docs/source/conf.py [hunk 1]
```diff
@@ -72,7 +72,7 @@
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 # This pattern also affects html_static_path and html_extra_path.
-exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
+exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store', '*requirements.txt']
```
Comment (deanwetherby) [inline docs/source/conf.py:75]: Added the symlinked requirements file to exclude patterns as sphinx was complaining about it.

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -5,6 +5,7 @@ on:
     branches-ignore:
       - development
       - master
+      - gh-pages
```
Comment (deanwetherby) [inline .github/workflows/ci.yaml:8]: No need for gh-pages (documentation) to run unit tests

**Comment by ThomasSchellenbergNextCentury** [comment]
We should add a link to the docs's main page into the repository's README, and/or update the links to specific pages in the [Documentation section](https://github.com/NextCenturyCorporation/MCS/tree/docs#documentation).

**Review by kdrumm — APPROVED** [review]
LGTM

**Review by brianpippin — APPROVED** [review]

**Pull request merged**